### ISSUE TITLE
walletd: --wallet-password-file?

### ISSUE THREAD

**Issue by KrzysztofHajdamowicz** [bcndev/bytecoin#98]
I'm running a production instance of bytecoin.
walletd requires to enter a password via stdin.
There's deprecated method to enter it via argv param.

Maybe there should be an option to specify a file with a password, so I can store passwd here and chmod 600 that file, so nobody else can touch it?

**Comment by ConnorBP** [comment]
I agree. I have been running into a similar problem. I'm likely going to make the modification my self very soon. I may set up an option to get the variable from the environment as well.

**Comment by ConnorBP** [comment]
feel free to use the code from this commit in my fork i just added ;)
https://github.com/cryptogems-currency/cryptogem/commit/90d1ca3b10feddbdf550a48f666e28997eb8fc9d
I have not tested the file one but it should work. The environment variables work perfectly however.

**Comment by bcndev** [comment]
@KrzysztofHajdamowicz Please have a look at the [subsection](https://github.com/bcndev/bytecoin/wiki/Running-Bytecoin-Services) in our wiki where we give a recipe for doing this right.

**Comment by KrzysztofHajdamowicz** [comment]
> @KrzysztofHajdamowicz Please have a look at the [subsection](https://github.com/bcndev/bytecoin/wiki/Running-Bytecoin-Services) in our wiki where we give a recipe for doing this right.

This isn't a solution for doing this right via systemd.
